packages/Menu/src/MenuItem.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { emotion } from '@leafygreen-ui/lib';
+import { colors } from '@leafygreen-ui/theme';
+import { cx } from 'emotion';
+import { menuGroup } from './MenuGroup';
+
+const { css } = emotion;
+
+const indentation = 16;
+
+const containerStyle = css`
+  min-height: 42px;
+  display: flex;
+  align-items: center;
+  padding-left: ${indentation}px;
+  text-decoration: none;
+  cursor: pointer;
+  color: ${colors.gray[1]};
+  text-decoration: none;
+
+  &:hover {
+    background-color: ${colors.gray[8]};
+    transition: background-color 200ms ease-in-out;
+  }
+
+  &:first-of-type ~ ${menuGroup.selector} {
+    border-top: 1px solid ${colors.gray[7]};
+  }
+
+  ${menuGroup.selector} + & {
+    border-top: 1px solid ${colors.gray[7]};
+  }
+
+  ${menuGroup.selector} ${menuGroup.selector} & {
+    padding-left: ${indentation * 2}px;
+  }
+`;
+
+const titleTextStyle = css`
+  font-size: 14px;
+  line-height: 16px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+  font-weight: normal;
+`;
+
+const descriptionTextStyle = css`
+  font-size: 12px;
+  margin-top: 2px;
+  margin-bottom: 2px;
+  font-weight: normal;
+`;
+
+const activeStyle = css`
+  background-color: ${colors.gray[8]};
+  position: relative;
+
+  &:before {
+    content: '';
+    position: absolute;
+    width: 3px;
+    top: 0;
+    bottom: 0;
+    left: -1px;
+    background-color: ${colors.green[3]};
+  }
+`;
+
+const disabledStyle = css`
+  background: ${colors.gray[8]};
+  color: ${colors.gray[4]};
+  cursor: not-allowed;
+`;
+
+interface MenuItemProps {
+  /**
+   * If supplied, MenuItem will be rendered inside of `a` tags.
+   */
+  href?: string;
+
+  /**
+   * Class name that will be applied to root MenuItem element.
+   */
+  className?: string;
+
+  /**
+   * Callback function to be executed when MenuItem is clicked.
+   */
+  onClick?: React.MouseEventHandler;
+
+  /**
+   * Main content displayed in MenuItem.
+   */
+  children?: React.ReactNode;
+
+  /**
+   * Description text displayed below title in MenuItem.
+   */
+  description?: string;
+
+  /**
+   * Determines whether or not the MenuItem is diabled.
```

Comment:
Typo `diabled` -> `disabled`

Reply:
I wish I could get a spellcheck package in vscode for my comments/markdown files

Reply:
😂